apps/parsley/src/components/LogRow/SkippedLinesRow/index.tsx

```diff
@@ -5,27 +5,25 @@ import { Body, BodyProps } from "@leafygreen-ui/typography";
 import { useLogWindowAnalytics } from "analytics";
 import Icon from "components/Icon";
 import { size } from "constants/tokens";
-import { ExpandedLines } from "types/logs";
+import { ExpandedLines, Range } from "types/logs";
 import { RootRowProps } from "../types";
 
 const SKIP_NUMBER = 5;
 
 interface CollapsedRowProps extends RootRowProps {
-  collapsedLines: number[];
   expandLines: (expandedLines: ExpandedLines) => void;
+  range: Range;
 }
 
-const CollapsedRow: React.FC<CollapsedRowProps> = ({
-  collapsedLines,
+const SkippedLinesRow: React.FC<CollapsedRowProps> = ({
```

Comment:
There are a few places in this file you reference CollapsedRow instead of SkippedLines row could we make these consistent